### DISCUSSION TITLE
How to Compile

### DISCUSSION THREAD

**Discussion by toboil-features** [mbrlabs/Lorien#309] [Q&A] [answered]
Essceptially as a user under Linux.

**Comment by wiretail** [comment, marked as answer]
I haven't tried it on linux (although the linux version builds happily in Windows) but the basic flow should be:

1. Install Godot 4.3 (Steam is easy, or https://godotengine.org/download/linux/ )
2. Clone the main branch of the repo somewhere
3. Start Godot and import ./lorien/ (the folder with project.godot)
4. In the top right, click "Run Project" (the left "play" button) and see if it works
5. Project -> Export... -> Linux  (set any settings you want and the out path)  ...Export Project.

...that's about it for any Godot app.  (I'm fairly new to Godot though, so others may have more info.)

**Reply by ncvescera** [reply]
> I haven't tried it on linux (although the linux version builds happily in Windows) but the basic flow should be:
> 
>     1. Install Godot 4.3 (Steam is easy, or https://godotengine.org/download/linux/ )
> 
>     2. Clone the main branch of the repo somewhere
> 
>     3. Start Godot and import ./lorien/ (the folder with project.godot)
> 
>     4. In the top right, click "Run Project" (the left "play" button) and see if it works
> 
>     5. Project -> Export... -> Linux  (set any settings you want and the out path)  ...Export Project.
> 
> 
> ...that's about it for any Godot app. (I'm fairly new to Godot though, so others may have more info.)

Is there a way to compile the project only from CLI ?
